connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -363,6 +363,8 @@ object CheckConnectJvmClientCompatibility {
       ), // developer API
       ProblemFilters.exclude[DirectMissingMethodProblem](
         "org.apache.spark.sql.Dataset.collectResult"),
+      ProblemFilters.exclude[DirectMissingMethodProblem](
```

Comment:
Can you also enable MiMa checks for observe by removing the problem filters for it?

Reply:
Done!